aries_cloudagent/DefaultMessageDelivery/OutboundMessageEmitter.py

```diff
@@ -0,0 +1,67 @@
+from aries_cloudagent.core.profile import Profile
```

Comment:
This package should use lower snake case. `default_message_delivery` instead of `DefaultMessageDelivery`.

Reply:
This file should also follow the lower snake case convention.

Reply:
good catch, I will update to lower snake case.